test/e2e_node/services/kubelet.go

```diff
@@ -194,17 +194,14 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		cmdArgs = append(cmdArgs,
 			systemdRun,
 			"-p", "Delegate=true",
+			"-p", "StandardError=file:"+framework.TestContext.ReportDir+"/kubelet.log",
```

Comment:
This works because we start the kubelet with the `--logtostderr` flag: https://github.com/kubernetes/kubernetes/pull/104047/files#diff-39ac8bcd746449ace559f6d09bd16a43bf207323181481899f906adb55b03c6fR222

Reply:
Gotchat. nice trick. thanks!